aspnetcore/security/anti-request-forgery.md

```diff
@@ -430,7 +430,7 @@ xhttp.onreadystatechange = function() {
 xhttp.open('POST', '/api/password/changepassword', true);
 xhttp.setRequestHeader("Content-type", "application/json");
 xhttp.setRequestHeader("X-CSRF-TOKEN", csrfToken);
-xhttp.send(JSON.stringify({ "newPassword": "ReallySecurePassword999$$$" }));
+xhttp.send(JSON.stringify({ "newPassword": $CREDENTIAL_PLACEHOLDER$ }));
```

Comment:
@blowdart see just above [this bookmark](https://review.docs.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-6.0&branch=pr-en-us-21851#angularjs) for the preceding code.
Does this sample need to be about changing a pw?
 Can't we change the preceding code to:

```
xhttp.open('POST', '/api/token/changetoken', true);
xhttp.setRequestHeader("Content-type", "application/json");
xhttp.setRequestHeader("X-CSRF-TOKEN", csrfToken);
xhttp.send(JSON.stringify({ "newToken": $CREDENTIAL_PLACEHOLDER$ }));
```

Reply:
@wadepickett  if @blowdart  accepts this change, you'll need to replace all occurrences of `ChangePassword` to `ChangeToken`.

Reply:
@blowdart , does the above pointed out by Rick look acceptable?

Reply:
Just changing to placeholder? Yup

Reply:
@blowdart, I want to remove password from the sample and replace it with `code`. The sample doesn't need to update a password.

Reply:
Fine by me then :)